src/features/create-quiz/create-quiz-second-card/create-quiz-second-card.tsx

```diff
@@ -182,11 +156,28 @@ export const CreateQuizSecondCard: React.FC = observer(() => {
     }
   }, [QuizStore.selectedQuestionID]);
 
+  useEffect(() => {
+    if (QuizStore.selectedQuiz) {
+      QuizStore.questions = QuizStore.selectedQuiz?.quiz.quizQuestions;
+
+      if (!QuizStore.isAnyEmptyQuestion) {
+        QuizStore.addEmptyQuestion();
+      } else {
+        QuizStore.addEmptyQuestion();
```

Comment:
The function call `addEmptyQuestion` does not need any check from what it seems. It could be called without the `if`.